demo/screens/Home.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import { Image, Button, StyleSheet, View, Text } from "react-native";
+import { createStackNavigator, createAppContainer } from "react-navigation";
+import { Account } from "tasit-sdk";
+import ListLands from "./ListLands";
+import LandClaim from "./LandClaim";
+import OnboardingHome from "./OnboardingHome";
+import EthereumQuestion from "./EthereumQuestion";
```

Comment:
This doesn't need to be imported from `components` a directory up? Is `components` inside screens right now? We'll want a `presentational` or `container` subdir too, I think.

When navigating around dirs gets complicated, it's nice to use @ package name and add tiny npm package.json's with just name to the folders to make them directly reference-able

Reply:
The components folders tree is:
screens/
components/container/
components/presentational/
Is it right?

Reply:
Nice tip.

Reply:
Yeah, that dir structure looks right.

Ah ok, so no `../components/*` was needed here because from this file we were just referencing other screens in the same dir.